buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -356,14 +356,10 @@ class ClusterFormationTasks {
                 'node.attr.testattr'           : 'test'
         ]
         int minimumMasterNodes = node.config.minimumMasterNodes.call()
-        if (minimumMasterNodes > 0) {
+        if (node.nodeVersion.before("7.0.0") && minimumMasterNodes > 0) {
             esConfig['discovery.zen.minimum_master_nodes'] = minimumMasterNodes
         }
-        if (minimumMasterNodes > 1) {
-            // don't wait for state.. just start up quickly
-            // this will also allow new and old nodes in the BWC case to become the master
-            esConfig['discovery.initial_state_timeout'] = '0s'
-        }
+        esConfig['discovery.initial_state_timeout'] = '0s'
```

Comment:
why remove the comment? It is still helpful to know why we set this here.

Reply:
Ok, reinstated in 6f5bb4345a2f2da5ebfbda4410697d223a3023a2.